src/main/java/com/maxi/calendar/model/Event.java

```diff
@@ -0,0 +1,71 @@
+package com.maxi.calendar.model;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name="event")
+@AllArgsConstructor @Getter @Setter
+public class Event {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	private int id;
+	
+	private Date day;
+	
+	@Column(name="begin_time")
+	private Date beginTime;
+	
+	@Column(name="end_time")
+	private Date endTime;
+	
+	private int status;
```

Comment:
Status should be an Enum with defined values like new, modified, accepted, canceled

Reply:
status enum created